apps-rendering/src/pagination.ts

```diff
@@ -0,0 +1,146 @@
+import { map, none, some, withDefault } from '@guardian/types';
+import type { Option } from '@guardian/types';
+import { pipe } from 'lib';
+import type { LiveBlock } from 'liveBlock';
+
+export type PageReference = {
```

Comment:
Here we can see a small divergence the `PageReference` interface compared to what's been defined in frontend (https://github.com/guardian/frontend/blob/main/common/app/model/LiveBlogCurrentPage.scala#L180) - here we don't define the `isArchivePage` boolean. I decided to remove this from the interface:
- In frontend, I think isArchivePage was used for appending the liveblog navigation container id to the navigation href links (https://github.com/guardian/frontend/blob/main/article/app/views/fragments/liveBlogNavigation.scala.html#L32)
- If I've understood this correctly, I feel like `isArchivePage` doesn't really describe why we're defining this property (I personally didn't find the name intuitive, even after understanding the purpose)
- In ar/dcr I think we can implement this same functionality (i.e. scrolling to a particular container id) in a slightly easier way (possibly inside the `Pagination` component itself) without the need for this extra property.